wagtail/admin/ui/side_panels.py

```diff
@@ -240,8 +240,9 @@ def get_context_data(self, parent_context):
 
 
 class PageStatusSidePanel(StatusSidePanel):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, parent_page=None, **kwargs):
         super().__init__(*args, **kwargs)
+        self.parent_page = parent_page
```

Comment:
It might be better to just use kwargs.pop, see other init methods for how this is done. 

This way we can probably avoid pulling out the kwarg in the function statement.